locales/languages.json

```diff
@@ -43,12 +43,6 @@
       "main": {},
       "params": {},
       "yourloops": {}
-    },
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏The countries should also be removed from the `CountryCodes` enum (see the file `country.model.ts`)